src/main/java/org/opensearch/agent/tools/PPLTool.java

```diff
@@ -329,4 +310,36 @@ private Map<String, String> extractFromChatParameters(Map<String, String> parame
         }
         return parameters;
     }
+
+    private String parseOutput(String llmOutput, String indexName) {
+        String ppl;
+        Pattern pattern = Pattern.compile("<ppl>((.|[\\r\\n])+?)</ppl>");
```

Comment:
Please add comments and examples to this regex expression.

Reply:
Already done/